butterknife-compiler/src/main/java/butterknife/compiler/ButterKnifeExtension.java

```diff
@@ -0,0 +1,9 @@
+package butterknife.compiler;
```

Comment:
I'm fairly sure this isn't possible because the classes are needed at compile time by the annotation processor.

In order to use this in an app, I think you'd have to add two extra modules.

1. Runtime custom view you want a butterknife bound listener (view code, and listener class annotation)
2. Compile time subclass of `ButterKnifeExtension`